test/runtime/runtime-go/Test_Go/Test_Address.go

```diff
@@ -0,0 +1,58 @@
+package Test_Go
+
+import (
+	"encoding/json"
+	runtimego "runtimego/target"
+	"testing"
+)
+
+func TestShouldbeAbleToSerializeModel(t *testing.T) {
+	address := runtimego.Address{
+		StreetName:           "Test Address 2",
+		HouseNumber:          2.0,
+		Marriage:             true,
+		Members:              nil,
+		ArrayType:            nil,
+		NestedObject:         nil,
+		AdditionalProperties: nil,
+	}
+
+	jsonBytes, err := json.Marshal(address)
+
+	if err != nil {
+		t.Fatalf("Failed to serialize Address to JSON: %v", err)
+	}
+	jsonStr := string(jsonBytes)
+
+	if jsonStr == "" {
+		t.Errorf("Serialize JSON is empty")
+	}
+}
+
+func TestShouldNotContainAdditionalPropertiesWhenSerialized(t *testing.T) {
+	address := runtimego.Address{
+		StreetName:           "Test Address 2",
+		HouseNumber:          2.0,
+		Marriage:             true,
+		Members:              nil,
+		ArrayType:            nil,
+		NestedObject:         nil,
+		AdditionalProperties: nil,
```

Comment:
@Devansh-Bhatt if you look at one of the other runtime test, do you mind using the same values as in there? 🙂